src/global/config.ts

```diff
@@ -19,11 +19,21 @@
 
 import packageJSON from '../../package.json';
 
-type AppConfig = { DOCS_URL_ROOT: string; UI_VERSION: string; REGION: string };
+type AppConfig = {
+	DOCS_URL_ROOT: string;
+	EGO_API_ROOT: string;
+	EGO_CLIENT_ID: string;
+	EGO_PUBLIC_KEY: string;
+	REGION: string;
+	UI_VERSION: string;
+};
 
 export const getAppConfig = (): AppConfig => {
 	return {
 		DOCS_URL_ROOT: process.env.NEXT_PUBLIC_DOCS_URL_ROOT || 'https://docs.icgc-argo.org/',
+		EGO_API_ROOT: process.env.EGO_API_ROOT || 'https://ego.dev.argo.cancercollaboratory.org',
```

Comment:
likely keep this as a localhost or something similar
dev.argo is still internal url (even if it's only dev) where as docs default is a publicly accessible site

Reply:
Yes in principal but in practice this is creating different URLs on the server and on the client for me

Reply:
I should be ok to be online tomorrow and you can tell me more about this because I don't understand.

I just mean keeping any info on our internal workings out of source control is good practice

Reply:
discussed on slack - for future ref : https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser